dropshot_endpoint/src/lib.rs

```diff
@@ -29,19 +29,22 @@ mod util;
 ///
 /// ```ignore
 /// #[endpoint {
-///     // Required fields
+///     // --- Required fields ---
+///     // The HTTP method for the endpoint
 ///     method = { DELETE | HEAD | GET | OPTIONS | PATCH | POST | PUT },
+///     // The path to the endpoint, along with path variables
 ///     path = "/path/name/with/{named}/{variables}",
 ///
-///     // Optional tags for the operation's description
+///     // --- Optional fields ---
+///     // Tags for the operation's description (default: empty)
 ///     tags = [ "all", "your", "OpenAPI", "tags" ],
-///     // Specifies the media type used to encode the request body
+///     // The media type used to encode the request body (default: application/json)
 ///     content_type = { "application/json" | "application/x-www-form-urlencoded" | "multipart/form-data" }
-///     // A value of `true` marks the operation as deprecated
+///     // True if the operation is deprecated (default: false)
 ///     deprecated = { true | false },
-///     // A value of `true` causes the operation to be omitted from the API description
+///     // True causes the operation to be omitted from the API description (default: false)
 ///     unpublished = { true | false },
-///     // Optional maximum request body size in bytes
+///     // Maximum request body size in bytes (default: server config default_request_body_max_bytes)
```

Comment:
this is good, but I think it would be nice to let people know that a CONSTANT would be fine here (the fact that any expression works is fine, but I think the interesting thing is that a constant works).

Reply:
I was trying to keep the comment short to be honest. I think the best thing to do here would be to use Markdown for all the configuration parameters, similar to the tag configuration section below.

Reply:
I've expanded the docs and added a clearer hint about being able to use constants.